packages/editor/src/package/index.ts

```diff
@@ -1,9 +1,12 @@
 export { SerloEditor, type SerloEditorProps } from './editor'
 export { SerloRenderer, type SerloRendererProps } from './serlo-renderer'
 
+export { defaultPlugins } from './config'
```

Comment:
What plugins are available can be set by passing `plugins?: (EditorPluginType | TemplatePluginType)[]` in the editor API. However, if none are passed `defaultPlugins` will be used. The edu-sharing integration wants to use `defaultPlugins` but remove `image` & `dropzone` and add some custom plugins. Then it could look like this: 
```typescript
      plugins={[
        ...defaultPlugins.filter(
          (plugin) =>
            plugin !== EditorPluginType.Image &&
            plugin !== EditorPluginType.DropzoneImage,
        ),
        EditorPluginType.EdusharingAsset,
        EditorPluginType.SerloInjection,
        EditorPluginType.TextAreaExercise,
      ]}
```

Reply:
If this is only an exception for Edusharing, could you please add a comment that this is exported only for Edusharing for this case?

If this is something that you imagine all integrations using, could you please document it in the Readme?